ci/build.sh

```diff
@@ -23,7 +23,8 @@ if [ -n "$GITHUB_TOKEN" ]; then
 fi
 
 # Check for invalid changes to generated yaml files (non-Tag builds)
-if [ -n "$GITHUB_TOKEN" ] && [ "$TRAVIS_EVENT_TYPE" == "pull_request" ]; then
+# Exclude branches that start with "revert-" to allow reverts
+if [ -n "$GITHUB_TOKEN" ] && [ "$TRAVIS_EVENT_TYPE" == "pull_request" ] && [[ ! "$TRAVIS_PULL_REQUEST_BRANCH" =~ ^revert- ]]; then
```

Comment:
Was the use of double brackets `[[ ]]` intentional? Not super familiar with bash but if it's not necessary here I vote we stick with single brackets for consistency

Reply:
Yes, the double brackets allow evaluating a regular expression in bash: https://unix.stackexchange.com/questions/306111/what-is-the-difference-between-the-bash-operators-vs-vs-vs
> Among its extended features, it can test whether a string matches a regular expression.